app/controllers/admin/users_controller.rb

```diff
@@ -7,6 +7,12 @@ def close_account
     redirect_to user_search_path, notice: t('admins.user_search.account_closing_scheduled', name: u.username)
   end
 
+  def lock_account
+    u = User.find(close_account_params)
+    u.lock_account!
+    redirect_to user_search_path, notice: t('admins.user_search.account_locking_scheduled', name: u.username)
```

Comment:
Line is too long. [109/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.